doc/api-examples/c/setup.md

```diff
@@ -0,0 +1,37 @@
+<!-- markdownlint-disable-file MD013 MD036 MD041 -->
+**Setup sample project**
+
+The C/C++ snippets require at least a simple project setup. Create it via:
+
+```bash
+# create project directory
+mkdir bluechi
+cd bluechi
+```
+
+**Installing dependencies**
+
+The C/C++ snippets use the [libsystemd](https://www.freedesktop.org/software/systemd/man/libsystemd.html) library for the D-Bus bindings. For compilation `gcc` and/or `g++` can be used. Install it via:
+
+```bash
+# on Ubuntu
+apt update
+apt install libsystemd-dev
+apt install build-essential 
+# on Fedora/CentOS
+dnf install systemd-devel
+dnf install gcc g++
```

Comment:
Wouldn't it be easier to install all requirements in a signle command?

```
apt install libsystemd-dev build-essential
```
```
dnf install gcc g++ systemd-devel
```

Reply:
Good point. I'll change it.